include/tvm/runtime/ndarray.h

```diff
@@ -84,6 +84,8 @@ class NDArray {
   bool same_as(const NDArray& other) const {
     return data_ == other.data_;
   }
+  /*! \return If both NDArray denote the same value */
```

Comment:
Just expose the GetDataSize function, the content memcmp is only available for CPU data and this function won't work in GPU

Reply:
It is fixed